### PR TITLE
feat:done with alert and button

### PR DESCRIPTION
**Pull request by tushargoyal812**
Added alert and button component

https://user-images.githubusercontent.com/69681896/152840836-b07dad8c-ec12-472f-8922-5dbd280e28b6.mp4

https://user-images.githubusercontent.com/69681896/152841018-b8c5c95b-77a4-4c70-9f17-d1c66c2426e4.mp4

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+@import url('/index.css');
+
+
+.alert-bg {
+    background-color: #fff;
+}
```
Comment (haarsh24) [inline components/alert/alert.css:6]: declare this color and use.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    background-color: rgb(99, 181, 231);
+}
+
+.success {
+    background-color: rgba(38, 189, 18, 0.726);
+}
+
+.error {
+    background-color: rgb(248, 64, 64);
+}
```
Comment (haarsh24) [inline components/alert/alert.css:39]: Use same color type throughout the code , If you are  using hex , use it eveywhere

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+.information-border {
+    border: 2px solid rgb(99, 181, 231);
+    color: rgb(99, 181, 231);
+}
+
+.success-border {
+    border: 2px solid rgba(38, 189, 18, 0.726);
+    color: rgba(38, 189, 18, 0.726);
+}
+
```
Comment (haarsh24) [inline components/alert/alert.css:55]: same here

File: components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    width: 70%;
+    padding: 2rem;
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    gap: 0.5rem;
+    flex-wrap: wrap;
+    background-color: var(--bg-color);
+    border-radius: 5px;
+}
```
Comment (haarsh24) [inline components/button/button.css:14]: same here

**Review by haarsh24 — CHANGES_REQUESTED** [review]
1. Declare color before using it .
2. use same color codes everywhere.

File: components/alert/alert.css (hunk 1 above)
Comment (yogesh-) [reply components/alert/alert.css:6]: use color variable

File: components/alert/alert.css (hunk 3 above)
Comment (yogesh-) [reply components/alert/alert.css:55]: use rem for consistency and good practice if possible.

**Review by yogesh- — APPROVED** [review]
Looks good to me. few minor changes suggested for the best practices.

**Pull request merged**